src/main/kotlin/app/revanced/patches/anytracker/misc/premium/annotations/UnlockPremiumCompatibility.kt

```diff
@@ -0,0 +1,9 @@
+package app.revanced.patches.anytracker.misc.premium.annotations
+
+import app.revanced.patcher.annotation.Compatibility
+import app.revanced.patcher.annotation.Package
+
+@Compatibility([Package("com.shervinkoushan.anyTracker")])
```

Comment:
I'm guessing the dev will continue to change his app, would it be a good idea to add a version constraint?

Reply:
I believe we can add the constraint when necessary, for now the fingerprint should work for future versions as well.